src/Serializer/TagCollectorInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Serializer;
+
+/**
+ * Interface for collecting cache tags during normalization.
+ *
+ * @author Urban Suppiger <urban@suppiger.net>
+ */
+interface TagCollectorInterface
+{
+    public function collect(array $context = []): void;
```

Comment:
This is quite nice actually! Can you document the context like this: https://github.com/api-platform/core/blob/00246aeb27f15dbc8eec5f41fa75a6cf7e64bce4/src/State/ProcessorInterface.php#L31

Reply:
Fixed in https://github.com/api-platform/core/pull/5758/commits/828ecb60ffeab01c34c9b6f922b208832d3a9d3d